src/overrides/venia-ui/components/ProductFullDetail/productFullDetail.module.css

```diff
@@ -0,0 +1,304 @@
+    composes: text-blue-midBlueShade from global;
+}
+
+.featureText {
+    color: black;
+}
+
+.imageCarousel {
+    grid-area: images;
+    grid-column: 1 / 2;
+    grid-row: 1 / fold;
+    composes: pl-7 from global;
+    composes: pr-7 from global;
+    composes: max-w-[640px] from global;
+    composes: mx-auto from global;
+    composes: my-0 from global;
+
+    composes: lg_m-0 from global;
+    composes: lg_px-0 from global;
+    composes: lg_py-sm from global;
+}
+
+.options {
+    grid-area: options;
+    composes: border-gray-grayCoose from global;
+}
+
+.formErrors {
+    composes: section;
+    grid-area: errors;
+}
+
+.productShippingActions {
+    composes: section;
+    composes: py-6 from global;
+    composes: border-none from global;
+    grid-area: quantity;
+}
+
+.shipping {
+    composes: w-full from global;
+    height: 11vh;
+    background-color: rgb(235, 235, 235);
+}
+
+.quantityTitle {
+    composes: title from '@magento/venia-ui/lib/components//ProductOptions/option.module.css';
+}
+
+.actions {
+    composes: gap-y-sm from global;
+    composes: grid from global;
+    composes: items-center from global;
+    composes: justify-items-center from global;
+    composes: section;
+    grid-area: cart;
+
+    composes: lg_border-b-0 from global;
+}
+
+.description {
+    composes: section;
+    width: 100%;
+    grid-column: 1 / 1 / 1 / 1;
+    grid-row: fold / span 1;
+
+    composes: lg_border-t from global;
+    composes: lg_m-0 from global;
+    composes: lg_pl-sm from global;
+    composes: lg_pr-sm from global;
+    composes: lg_self-stretch from global;
+}
+
+.descriptionTitle {
+    composes: sectionTitle;
+}
+
+
+.detailsPageBuilder {
+    composes: my-0 from global;
+    composes: mx-sm from global;
+    composes: px-0 from global;
+    composes: py-sm from global;
+    grid-column: 1 / span 2;
+}
+
+.detailsPageBuilderList {
+    list-style: none;
+}
+
+:global .shippingBlock {
+    flex-direction: row !important;
+    padding: 20px 50px !important;
+    width: 100% !important;
+}
+
+:global .paidShippingBlock {
+    border-bottom: 1px solid #DFE1E2;
+}
+
+:global .shippingText {
+    margin-top: 10px !important;
+    margin-left: 30px !important;
+}
+
+:global .priceText {
+    margin-left: -125px !important;
+    margin-top: 38px !important;
+}
+
+
+@media (min-width: 1024px) {
+    .details {
+        align-self: stretch;
+        border-top-width: 1px;
+        margin: 0;
+        padding-left: 1.5rem;
+        padding-right: 1.5rem;
+    }
+
+    .productName {
+        font-size: 2rem;
+    }
+
+    .productPrice {
+        font-size: 2rem;
+    }
+    
+    .imageCarousel {
+        padding-left: 0px;
+        padding-right: 0px;
+    }
+
+    :global .shippingBlock {
+        flex-direction: row !important;
+        padding: 20px 70px !important;
+        width: 100% !important;
+    }
+
+    :global .paidShippingBlock {
+        border-bottom: 0px;
+        border-right: 1px solid #DFE1E2;
+    }
```

Comment:
We still should use tailwind classes instead of pure css. in this case as we can't use composes, we just use @apply